src/components/form.js

```diff
@@ -15,18 +15,26 @@
           filter,
           formErrorMessage,
           formSuccessMessage,
+          redirect,
         } = options;
 
         const formRef = React.createRef();
 
         const empty = children.length === 0;
         const isDev = B.env === 'dev';
         const isPristine = empty && isDev;
+        const redirectTo =
+          B.env === 'prod' && redirect && B.useEndpoint(redirect);
 
         return (
           <Action actionId={actionId}>
             {(callAction, { data, loading, error }) => (
               <>
+                {(() => {
+                  if (data && redirectTo) {
+                    window.location.hash = redirectTo;
```

Comment:
Please use `useHistory` instead of window.location, like explained here: https://github.com/bettyblocks/cli/wiki/Component-Helpers#usehistory and here https://github.com/bettyblocks/cli/wiki/Component-Helpers#buseendpoint